### PR TITLE
Feature/travis readme files

### PR DESCRIPTION
**Pull request by nyashabryan**
The travis file and README file still need to be edited by this is a start.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+momkhulu
+=============================
+
+This is a hospital resource allocation tool. Provides an API which can be
+used with RapidPro. Uses WebSockets for real-time page updates.
```
Comment (KaitCrawford) [inline README.rst:5]: WebSockets or Django Channels? or are they the same thing?

**Comment by KaitCrawford** [comment]
Small question but it looks good. The readme especially is good

File: README.rst (hunk 1 above)
Comment (nyashabryan) [reply README.rst:5]: So WebSockets is the transport protocol which is used by Django Channels.

**Review by KaitCrawford — APPROVED** [review]
:+1:

**Pull request merged**